src/components/Button/Button.tsx

```diff
@@ -1,3 +1,77 @@
-import { Button } from "theme-ui"
+/** @jsxRuntime classic /
+/** @jsx jsx */
+import { Button as ThemeUIButton, jsx, SxStyleProp } from "theme-ui"
+import { MouseEvent, ButtonHTMLAttributes, forwardRef } from "react"
+import { shade } from "@theme-ui/color"
+
+export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  children?: string
+
+  variant?: string
+
+  color?: string
```

Comment:
I think the point is to receive the color name, not the actual hex code. If that's the case, maybe it would be better if it was typed as such, like `"primary" | "secondary" | ...` and so on

Reply:
that's correct @giubatt, I intend to use the values from the theme, so it should be color itself, not className

Reply:
Maybe we should come back to this after I work on the colors scheme to make sure we can add typings to the color prop